asset_tracker_app/lib/utils/constants/theme/constant_icons.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:asset_tracker_app/utils/constants/theme/constant_sizes.dart';
+import 'package:flutter/material.dart';
+
+class ConstantIcons {
+  static Icon getSplashScreenIcon(BuildContext context) {
+    return Icon(
+      Icons.inventory,
+      size: ConstantSizes.iconXXXL,
+      color: Theme.of(context).primaryColor,
+    );
+  }
+
+  static Icon getOnboardPagesIcons(IconData icon, BuildContext context) {
```

Comment:
böyle statik bir fonksiyon oluşturmak yerine, CustomIcon adında widget oluşturabilirsin. Dışarıdan icon data alır. opsiyonel olarak size değeri iconXXL olur. Widget yapabileceğin senaryolarda statik metot yapmanı pek önermem.